### PR TITLE
fixing tests after moving to dynflow for repo creation/deletion

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
requires   https://github.com/Dynflow/dynflow/pull/106  (and https://github.com/iNecas/dynflow/tree/stub-plan-action, since cv-rework-dynflow-publish already requires it)

File: test/glue/pulp/repository_test.rb [hunk 1]
```diff
@@ -68,8 +63,8 @@ def self.before_suite
   end
 
   def self.after_suite
-    @@fedora_17_x86_64.destroy_repo
-    VCR.eject_cassette
+    ::ForemanTasks.sync_task(::Actions::Pulp::Repository::Destroy, :pulp_id => @@fedora_17_x86_64.pulp_id)
+        VCR.eject_cassette
```
Comment (iNecas) [inline test/glue/pulp/repository_test.rb]: Maybe worth extracting the `sync_task` call into some helper method in `GluePulpRepoTestBase`

Comment (iNecas) [inline test/glue/pulp/repository_test.rb]: The CVR.eject_casette has some odd indentation

**Comment by iNecas** [comment]
Merged, thx

**Pull request closed without merging**